astropy/modeling/fitting.py

```diff
@@ -1674,7 +1687,18 @@ def __call__(self, model, *args, **kwargs):
                 AstropyDeprecationWarning,
```

Comment:
@nden I think the deprecation warning is still here

Reply:
Hm, I don't see it on main. Or am confused and somehow looking at the wrong file.

Reply:
See here:

https://github.com/astropy/astropy/blob/11182c3243bf3c55d812d0ec927ab4cb5e7e024b/astropy/modeling/fitting.py#L1724